chat-plugins/mafia.js

```diff
@@ -1029,6 +1037,28 @@ exports.commands = {
 			game.updateHost();
 		},
 
+		subhost: function (target, room, user) {
+			if (!this.canTalk()) return;
+			if (!target) return this.parse('/help mafia subhost');
+			this.splitTarget(target);
+			let targetUser = this.targetUser;
+			if (!targetUser || !targetUser.connected) return this.errorReply(`The user "${this.targetUsername}" was not found.`);
+			if (!room.users[targetUser.userid]) return this.errorReply(`${targetUser.name} is not in this room, and cannot be hosted.`);
+			if (!user.can('mute', null, room)) return this.errorReply(`/mafia subhost - Access denied.`);
+			if (Object.keys(room.game.players).includes(targetUser.userid)) return this.errorReply(`You cannot subhost to a user in the game.`);
```

Comment:
check `room.game.dead` as well (treestumps, restless spirits, they can also be revived)